rsql-jpa/src/main/java/io/github/perplexhub/rsql/SortUtils.java

```diff
@@ -64,16 +64,16 @@ private static Order sortToJpaOrder(final String[] parts, final SortSupport sort
                         sortSupport.getJoinHints(), sortSupport.getProcedureWhiteList(),
                         sortSupport.getProcedureBlackList());
 
+        final boolean ic = parts.length > 2 && "ic".equalsIgnoreCase(parts[2]);
```

Comment:
We probably don't need to add an extra parameter for changing jsonb function.
Casting jsonb extract path to text in JPA should be sufficient, and we can remove unchecked cast.
`@SuppressWarnings("unchecked")`